src/evict/evict_page.c

```diff
@@ -271,7 +273,8 @@ __evict_child_check(WT_SESSION_IMPL *session, WT_REF *parent)
  */
 static int
 __evict_review(
-    WT_SESSION_IMPL *session, WT_REF *ref, int exclusive, int *inmem_splitp)
+    WT_SESSION_IMPL *session, WT_REF *ref, int exclusive, int *inmem_splitp,
```

Comment:
Generally with line-splitting the last line should be the longest, so you'd move most of the parameters onto the new line here, rather than just the final one.